src/seedu/addressbook/data/person/Block.java

```diff
@@ -0,0 +1,19 @@
+package seedu.addressbook.data.person;
+
+public class Block {
+	
+	private String block;
+	
+	public Block(String value) {
+		this.block = value;
+	}
+	
+	public String getBlock() {
+		return block;
+	}
+	
+	public String toString() {
+		return this.getBlock();
+	}
+
+}
```

Comment:
The block class (and the other 3 new classes) are a little bare bone :-). Consider adding validation as an added functionality.